### PR TITLE
refactor(playsounds): use regex instead of elif chaining

### LINKED ISSUE #96: Fix If Elif spam in playsounds code.
This is, safe to say, awful.

https://github.com/mmattDonk/AI-TTS-Donations/blob/e3512f69818c4e4c36dfc877bc1033f4496aca24/bot.py#L116-L511

### PR DESCRIPTION
**Pull request by 12beesinatrenchcoat**
> - makes an array (playsounds) containing all the .wav files in the /playsounds/ directory
> - matches message with regex ^\(\d+\)$
> - checks if sound exists

resolves #96.

## Description
Because `if else` chaining hurts my soul.

## How Has This Been Tested
Ran two tests:
> (100)  

Should result in no sound being played.

> (100) || ninja: wow  

Should result in "ninja: wow" going through to Uberduck.ai, and nothing else.

## Checklist

- [x] Changes follow the **CONTRIBUTING**.md guidelines.
- [x] Update necessary documentation accordingly.
- [x] Check issues and pull requests first. You don't want to duplicate effort.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Changes unreleased.
 
 - dependencies:
   - remove duplicate entry ([89f4237](https://github.com/mmattDonk/AI-TTS-Donations/commit/89f42374c3466bc027efcdb4bdae10226df063b1))
+- playsounds:
+  - remove `elif` chaining ([#115](https://github.com/mmattDonk/AI-TTS-Donations/pull/115))
```
Comment (mmattbtw) [inline CHANGELOG.md:12]:
```suggestion
```

remove this entry because I have a github action that automatically adds changelog entries

**Review by mmattbtw — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (12beesinatrenchcoat) [reply CHANGELOG.md:12]: ah kk

File: bot.py [hunk 2]
```diff
@@ -112,404 +118,22 @@ def request_tts(message: str, failed: Optional[bool] = False):
...
-            continue
-        # ----------- End of Playsounds -----------
+
+        playsound = re.match(r"^\(\d+\)$", message)
+
+        if playsound:
+            i = int(playsound.group()[1:-1]) - 1
+
+            if i < 1 or i > (len(playsounds) - 1):
+                print(f"sound {i} does not exist. it will not be played.")
```
Comment (mmattbtw) [inline bot.py:131]:
```suggestion
                log.info(f"sound {i} does not exist. it will not be played.")
```

**Review by mmattbtw — CHANGES_REQUESTED** [review]
Also `(1)` doesn't seem to work for me:
image

Using this string: `spongebob: test || (1) || drake: test || (2) || kanye-west-rap: test || drake: test` or this string: `(1) || (2) || (3)`

**Comment by 12beesinatrenchcoat** [comment]
oh yeah i did some wonky stuff with numbers didn't i

**Review by mmattbtw — APPROVED** [review]
tested locally, all good 👍 thanks for your continued contributions!

**Comment by mmattbtw** [comment]
@all-contributors add @12beesinatrenchcoat for doc, and code

**Pull request merged**